deploy.sh

```diff
@@ -77,7 +77,14 @@ parse_args() {
 
 	#if no user identity is already set in the current git environment, use this:
 	default_username=${GIT_DEPLOY_USERNAME:-deploy.sh}
-	default_email=${GIT_DEPLOY_EMAIL:-}
+	
+	if [ -z "$GIT_DEPLOY_EMAIL" ]
+	then
+		#if no default email, we get the last commits author
+		default_email=$(git log -n 1 --format="%ae")
+	else
+		default_email=${GIT_DEPLOY_EMAIL:-}
+	fi
```

Comment:
Isn't this possible?

```suggestion
 	default_email=${GIT_DEPLOY_EMAIL:$(git log -n 1 --format="%ae")} 
```